producer-rest-core/src/main/scala/za/co/absa/spline/producer/rest/controller/ExecutionProducerController.scala

```diff
@@ -134,7 +134,7 @@ class ExecutionProducerController @Autowired()(
         ]
       """
   )
-  def executionEvent(@RequestBody execEvents: Array[ExecutionEvent]): Future[Array[String]] = {
+  def executionEvent(@RequestBody execEvents: Array[ExecutionEvent]): Future[Unit] = {
```

Comment:
We don't return anymore the ids that were inserted ?

Reply:
The reasoning is this - what would client do with the events that weren't stored for whatever reason? What are those reasons for having some event stored and some don't? The whole thing is executed transactionally anyway. So I in case of a fatal error nothing is going to be persisted. But if there is logical error in some events (e.g events pointing non-existing executions) then again, why could happen? Spline Agent always persists execution plan and event sequentially. Async persistence might happen in case of streaming, reading from Kafka etc. But on one side we don't support that type of API yet, and on the other hand why would one store the event in batches if there is a Kafka in disposition. So in overall, it feels that at this stage of development taking care of those partially stored events is a bit of over-engineering. So I'd rather remove this actually unused piece and focus more on making the core stuff done.

Reply:
ok ok